daemon/metrics.go

```diff
@@ -140,7 +140,7 @@ func (p metricsPlugin) sock() string {
 }
 
 func (p metricsPlugin) sockBase() string {
-	return filepath.Join(p.BasePath(), "run", "balena")
+	return filepath.Join(p.BasePath(), "run", "balena-engine")
```

Comment:
"metrics plugin"...? The string "run" here reminded me of '/var/run/balena', hence renaming to 'balena-engine', but I am not sure about this change.